modules/transport-netty4/build.gradle

```diff
@@ -42,13 +42,6 @@ dependencies {
     compile "io.netty:netty-transport:4.1.6.Final"
 }
 
-integTest {
-    includePackaged = true
-    cluster {
-        numNodes = 2
```

Comment:
I understand that; we use to run the regular REST tests here with two nodes. I'm saying we should still run the regular REST tests with two nodes somewhere, probably as part of the regular distribution integration test. This seemed the best place to leave that thought. :smile: